Update 1-Penetration_Testing_Methodologies.md (#506)
OWASP/wstg: document/3-The_OWASP_Testing_Framework/1-Penetration_Testing_Methodologies.md

### PCI DSS Penetration Testing Guidance

> The PCI DSS Penetration testing guideline provides the methodology 

Reviewer: This is no longer a sentence?

- Application-layer testing
- Network-layer tests for network and OS

[PCI DSS Penetration Test Guidance](https://www.pcisecuritystandards.org/documents/Penetration_Testing_Guidance_March_2015.pdf)

## Penetration Testing Framework

> The Penetration Testing Framework (PTF) provides comprehensive hands-on penetration testing guide. It also lists usages of the security testing tools in each testing category. The major area of penetration testing includes -

Reviewer 2: ```suggestion
The Penetration Testing Framework (PTF) provides comprehensive hands-on penetration testing guide. It also lists usages of the security testing tools in each testing category. The major area of penetration testing includes:
```

- Application-layer testing
- Network-layer tests for network and OS

[PCI DSS Penetration Test Guidance](https://www.pcisecuritystandards.org/documents/Penetration_Testing_Guidance_March_2015.pdf)

> 

Reviewer: ```suggestion
```

## Penetration Testing Framework (PTF)

The Penetration testing framework provides comprehensive hands-on penetration testing guide. It also lists usages of the security testing tools in each testing category. The major area of penetration testing includes -

> - Network Foot-printing (Reconnaissance)

Reviewer: ```suggestion
- Network Footprinting (Reconnaissance)
```

[Penetration Testing Framework](http://www.vulnerabilityassessment.co.uk/Penetration%20Test.html)

## Technical Guide to Information Security Testing and Assessment (NIST 800-115)

> Technical Guide to Information Security Testing and Assessment was published by NIST U.S. Department of Commerce. There are some assessment techniques suggested by the NIST 800-115 listed below.

Reviewer: ```suggestion
Technical Guide to Information Security Testing and Assessment was published by NIST, it includes some assessment techniques listed below.
```

- Review Techniques
- Target Identification and Analysis Techniques
- Target Vulnerability Validation Techniques 
- Security Assessment Planning
- Security Assessment Execution
- Post-Testing Activities

> The NIST 800-115 can be accessed [here]( https://csrc.nist.gov/publications/detail/sp/800-115/final)

Reviewer: ```suggestion
The NIST 800-115 can be accessed [here](https://csrc.nist.gov/publications/detail/sp/800-115/final)
```

## Information Systems Security Assessment Framework (ISSAF)

> The ISSAF which was previously supported by Open Information Systems Security Group (OISSG) and was not an active community. However, it provides the following specific scenario-based security assessment.

Reviewer: If it's unsupported/inactive I'd rather drop it. The SourceForge content [hasn't been updated since 2005 and one of the assets is marked as a potential malware source](https://sourceforge.net/projects/isstf/files/issaf%20document/issaf0.1/) (likely just a web backdoor or something, but I still wouldn't trust it or expect anyone else to).

Author: Agreed.  In that case, we may remove the whole section? "ISSAF"　？

Reviewer: Yes

Author: Should I edit it & submit another PR or you can help me to remove the section?

Reviewer: You can edit the file further and push it again.

- Database Security Assessment
- WLAN Security Assessment
- Switch Security Assessment
- Router Security Assessment
- Firewall Security Assessment
- Intrusion Detection System Security Assessment
- VPN Security Assessment
- Storage Area Network (SAN) Security Assessment

Refer to the ISSAF Chapter 7 for the listed security assessment above.
[ISSAF References](https://sourceforge.net/projects/isstf/)

## Open Source Security Testing Methodology Manual (OSSTMM)

> OSSTMM is a methodology to test the operational security of physical locations, workflow, human security testing, physical security testing, wireless security testing, telecommunication security testing, data networks security testing and compliance. OSSTMM can be supporting reference of IOS 27001 instead of a hands-on or technical application penetration testing guide. 

Reviewer: You've somehow re-introduced an error here. It's not "IOS 27001", it's "ISO 27001".

OSSTMM includes the following key sections:
- Security Analysis
- Operational Security Metrics
- Trust Analysis

> - Work Flow.

Reviewer: ```suggestion
- Work Flow
```

For consistency.

- Compliance Regulations
- Reporting with the STAR (Security Test Audit Report)

[Open Source Security Testing Methodology Manual](https://www.isecom.org/OSSTMM.3.pdf)

> 

Reviewer: ```suggestion
```

Reviewer: Second blank is un-necessary.

## References

- [PCI Data Security Standard - Penetration TestingGuidance](https://www.pcisecuritystandards.org/documents/Penetration-Testing-Guidance-v1_1.pdf)

> - [PTES PenTest Standard](http://www.pentest-standard.org/index.php/Main_Page)

Reviewer: The title only needs to say Penetration Test once, pick "PTes" or "PenTest"

Author: May I use "PTES"?  Since it's the original logo "PTES" of the pentest-standard Thanks.

Reviewer: Sure "PTes Standard" seems fine.

- [PTES PenTest Standard](http://www.pentest-standard.org/index.php/Main_Page)
- [Open Source Security Testing Methodology Manual (OSSTMM)](http://www.isecom.org/research/osstmm.html)
- [NIST - SP 800-115](https://csrc.nist.gov/publications/detail/sp/800-115/final)

> - [HIPAA 2012](http://csrc.nist.gov/news_events/hiipaa_june2012/day2/day2-6_kscarfone-rmetzer_security-testing-assessment.pdf)

Reviewer: Why was this dropped?

Author: I re-organized these two references in Line 153/154. These two were not dropped.

- [Technical Guide to Information Security Testing and Assessment NIST SP 800-115](https://csrc.nist.gov/publications/detail/sp/800-115/final)
- [HIPAA Security Testing Assessment 2012](http://csrc.nist.gov/news_events/hiipaa_june2012/day2/day2-6_kscarfone-rmetzer_security-testing-assessment.pdf)

- [Technical Guide to Information Security Testing and Assessment NIST SP 800-115](https://csrc.nist.gov/publications/detail/sp/800-115/final)
- [HIPAA Security Testing Assessment 2012](http://csrc.nist.gov/news_events/hiipaa_june2012/day2/day2-6_kscarfone-rmetzer_security-testing-assessment.pdf)
- [Penetration Testing Framework 0.59](http://www.vulnerabilityassessment.co.uk/Penetration%20Test.html)
- [OWASP Mobile Security Testing Guide](https://owasp.org/www-project-mobile-security-testing-guide/)
- [Security Testing Guidelines for Mobile Apps](https://owasp.org/www-pdf-archive/Security_Testing_Guidelines_for_mobile_Apps_-_Florian_Stahl%2BJohannes_Stroeher.pdf)

> - [Kali](https://www.kali.org/)

Reviewer: Why was this dropped?

Author: It's not dropped. I updated with "[Kali Linux](https://www.kali.org/)" in line 158

> - [Information Systems Security Assessment Framework ISSTF](https://sourceforge.net/projects/isstf/files/issaf%20document/issaf0.1/)

Reviewer: ```suggestion
```
This was linked back at L124. Plus the naming here is inconsistent.
